server/src/main/java/org/solarbank/server/dto/CalculateResult.java

```diff
@@ -0,0 +1,40 @@
+package org.solarbank.server.dto;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import javax.money.CurrencyUnit;
+import javax.money.Monetary;
+import javax.money.MonetaryAmount;
+import java.util.Map;
+import lombok.Data;
+import org.javamoney.moneta.Money;
+
+@Data
+@JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
+public class CalculateResult {
+    private double energyGenPerYear;
+    private Map<String, Double> energyGenPerMonth;
+    private SavingsPerYear savingsPerYear;
+
+    @JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
+    public static class SavingsPerYear {
+        private CurrencyUnit currencyCode;
+        private MonetaryAmount amount;
```

Comment:
used CurrencyUnit and MonetaryAmount for this attributes to handle money/currency. Not sure about the getters/setters below?

Reply:
Removed getters and setters, setting CurrencyUnit and MontaryAmount directly. Also added serializers so only values returned to client